module/apmgormv2/driver/postgres/init.go

```diff
@@ -0,0 +1,42 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// Package apmpostgres imports the gorm postgres dialect package,
+// and also registers the lib/pq driver with apmsql.
+package apmgpostgres
+
+import (
+	"go.elastic.co/apm/module/apmsql"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+
+	_ "go.elastic.co/apm/module/apmsql/pq" // register lib/pq with apmsql
```

Comment:
Apparently gorm v2 uses pgx by default: https://github.com/go-gorm/postgres/blob/master/postgres.go

Maybe don't add the postgres dialect for now? WDYT?

Reply:
Yes, Removing Postgres dialect for now